code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -711,6 +711,21 @@
 	icon_state = "female_blackalt"
 	gender = FEMALE
 
+/datum/sprite_accessory/underwear/female_usastripe
+	name = "Ladies Freedom"
+	icon_state = "female_assblastusa"
+	gender = FEMALE
+
+/datum/sprite_accessory/underwear/female_uk
+	name = "Ladies UK"
+	icon_state = "female_uk"
+	gender = FEMALE
+
+/datum/sprite_accessory/underwear/female_commie
+	name = "Ladies Commie"
+	icon_state = "female_commie"
+	gender = FEMALE

```

Comment:
guys already got their boxers, besides, it'd be a little inconsistent if only these were neutral and the rest of the sets weren't.